packages/react/src/Button/styles.ts

```diff
@@ -18,6 +18,7 @@ export const getVariantStyles = (variant: VariantType = 'default', theme?: Theme
       '&:disabled': {
         color: 'primer.fg.disabled',
         borderColor: `var(--button-default-borderColor-disabled, ${theme?.colors.btn.border})`,
+        backgroundColor: `var(--button-default-bgColor-disabled, ${theme?.colors.input.disabledBg})`,
```

Comment:
Are we sure `theme.colors.input.disabledBg` is the right path for the fallback? It's rendering as `undefined` in the Jest snapshot tests.

Reply:
It looks like the border color above is also undefined, so I assumed there's something up with the test